frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -212,28 +232,34 @@ const AdminShoutouts: React.FC = () => {
 
   return (
     <div>
-      <Form className={styles.shoutoutForm}>
-        <h2>Filter shoutouts:</h2>
-        <Form.Group width="equals">
-          <ChooseDate dateField={earlyDate} dateFunction={setEarlyDate} />
-          <ChooseDate dateField={lastDate} dateFunction={setLastDate} />
-          <Button.Group className={styles.buttonGroup}>
-            <ButtonPiece shoutoutList={displayShoutouts} buttonText={'ALL'} />
-            <ButtonPiece
-              shoutoutList={displayShoutouts.filter((shoutout) => shoutout.hidden)}
-              buttonText={'HIDDEN'}
-            />
-            <ButtonPiece
-              shoutoutList={displayShoutouts.filter((shoutout) => !shoutout.hidden)}
-              buttonText={'PRESENT'}
-            />
-          </Button.Group>
-        </Form.Group>
-      </Form>
-      <div className={styles.shoutoutsListContainer}>
-        <ListTitle />
-        <DisplayList />
-      </div>
+      {loading ? (
+        <Loader active inline="centered" />
+      ) : (
+        <div>
+          <Form className={styles.shoutoutForm}>
+            <h2>Filter shoutouts:</h2>
+            <Form.Group width="equals">
+              <ChooseDate dateField={earlyDate} dateFunction={setEarlyDate} />
+              <ChooseDate dateField={lastDate} dateFunction={setLastDate} />
+              <Button.Group className={styles.buttonGroup}>
+                <ButtonPiece shoutoutList={displayShoutouts} buttonText={'ALL'} />
+                <ButtonPiece
+                  shoutoutList={displayShoutouts.filter((shoutout) => shoutout.hidden)}
+                  buttonText={'HIDDEN'}
+                />
+                <ButtonPiece
+                  shoutoutList={displayShoutouts.filter((shoutout) => !shoutout.hidden)}
+                  buttonText={'PRESENT'}
+                />
+              </Button.Group>
+            </Form.Group>
+          </Form>
+          <div className={styles.shoutoutsListContainer}>
```

Comment:
Can you instead wrap `<div className={styles.shoutoutsListContainer}></div>` with the ternary with Loader? That way we can see a least some part of the page before everything loads in.